Library/Homebrew/brew.sh

```diff
@@ -12,10 +12,28 @@ then
   then
     export LC_ALL="en_US.UTF-8"
   else
-    export LC_ALL="C.UTF-8"
+    utf8_locales=$(locale -a | grep -e \.utf8 -e \.UTF-8)
+    if [[ -n $utf8_locales && 
+         ($utf8_locales =~ (^|[^a-zA-Z0-9_])(C|en_US)\.(utf8|UTF-8) ||
+          $utf8_locales =~ ^[a-zA-Z0-9_.]*) ]]
+    then
+      export LC_ALL=$BASH_REMATCH
+    elif [[ -n $(locale -a | grep -w C) ]]
+    then
+      export LC_ALL=C
+    else
+      echo "Error: Could not find usable locale" >&2
+    fi
   fi
 fi
 
+# Configure TTY to work with UTF8
+# https://unix.stackexchange.com/a/476671/222638
+if [[ $(stty -a | grep -o '.iutf8') == -iutf8 ]]
```

Comment:
Why is this needed now and wasn't before?

Reply:
Also `stty: 'standard input': Inappropriate ioctl for device` from the GitHub Actions output looks like a new error.

Reply:
Apparently, TTYs can support (or not) utf8 as well, so this step enables that support. I'm not sure this is necessary, so I'm going to remove it from this PR. We can look into this further if someone reports a problem.